ContentPatcher/IManagedValue.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace ContentPatcher
+{
+    /// <summary>A set of parsed values linked to the Content Patcher context. These values are <strong>per-screen</strong>, so the result depends on the screen that's active when calling the members.</summary>
```

Comment:
I suggest changing "_a set of parsed values linked to..._" to something like "_a parsed string which may contain Content Patcher tokens linked to..._" for clarity.